### PR TITLE
GitHub actions fix

### PR DESCRIPTION
**Pull request by xyloflake**

### CODE REVIEW

**Comment by xyloflake** [comment]
So apparently, building macos is only possible on macos
image

**Comment by xyloflake** [comment]
@staniel359 @gamersi ready for review, done with the builds. 

Check out the actions build 
https://github.com/staniel359/muffon/actions/runs/8859069681

**Review by gamersi — DISMISSED** [review]
LGTM

**Review by gamersi — APPROVED** [review]
LGTM

File: electron-builder.json
Comment (staniel359) [inline electron-builder.json:1]: @xyloflake What to do with x64/arm64?

Comment (xyloflake) [reply electron-builder.json:1]: It's not working. If you set arch in target, macos build fails with distribution.xml no such file error. To build on x64 use "pnpm release --x64" and gh actions by default has arm64 so it's automatically done.

Comment (xyloflake) [reply electron-builder.json:1]: We should leave this as it is

Comment (gamersi) [reply electron-builder.json:1]: It’s arm64 by default because the runners are arm. Can you research if they are all arm?

Comment (xyloflake) [reply electron-builder.json:1]: They are all arm yes

Comment (xyloflake) [reply electron-builder.json:1]: @staniel359 https://github.com/electron-userland/electron-builder/issues/6015

Comment (staniel359) [reply electron-builder.json:1]: @xyloflake As long as we have it built, config doesn't matter.

**Review by staniel359 — APPROVED** [review]

File: electron-builder.json
Comment (xyloflake) [reply electron-builder.json:1]: @staniel359 and this'll not let it build so ig this is the only way around

Comment (xyloflake) [reply electron-builder.json:1]: @staniel359 it straight up fails if you don't remove the arch section

**Pull request merged**